### PR TITLE
Showing Customized Tag on Shared Word Card

### PR DESCRIPTION
**Pull request by akanex115**
# Background
https://github.com/ajktown/wordnote/issues/164

## TODOs
- [x] Implement unclickable tags for shared word card

## Checklist (Developers)
- [x] Make this PR as a draft first
- [x] Title is checked
- [x] Background & TODOs are filled
- [x] Assignee is set
- [x] Labels are set
- [x] Project is created & linked, if multiple PRs are associated to this PR
- [x] Appropriate issue(s) is(are) linked to this PR under `development`
- [x] `yarn inspect` is run
- [x] `TODOs` are handled and checked
- [x] Final Operation Check is done
- [x] Make this PR as an open PR

## Checklist (Reviewers)
Please copy and paste the following into your review, ensuring each checkbox is marked as checked
```
- [ ] Check if there are any other missing TODOs that are not yet listed
- [ ] Review Code
- [ ] Every item on the checklist has been addressed accordingly
- [ ] If `development` is associated to this PR, you must check if every TODOs are handled
```

### CODE REVIEW

File: src/components/molecule_word_card/index.shared.tsx [hunk 1]
```diff
@@ -34,35 +36,43 @@ const WordCardShared: FC<Props> = ({ wordId }) => {
     [wordId],
   )
 
-  if (sharedWord === undefined) return <WordCardSkeleton />
-  if (sharedWord === null || sharedWord.word === null)
+  if (sharedData === undefined) return <WordCardSkeleton />
```
Comment (mlajkim) [inline src/components/molecule_word_card/index.shared.tsx:39]: rollback the variable name from `sharedData` to `sharedWord` and create an issue with refactor that fix the naming problem we have

File: src/pages/share/index.tsx [hunk 2]
```diff
@@ -23,10 +23,10 @@ const SharePage: FC = () => {
     )
   return (
     <StyledCentered>
-      <Typography variant="h5">
+      <Typography variant="h5" style={{ border: `3px solid yellow` }}>
```
Comment (mlajkim) [inline src/pages/share/index.tsx:26]: Please remove these after test is done

File: src/pages/share/index.tsx [hunk 3]
```diff
@@ -23,10 +23,10 @@ const SharePage: FC = () => {
...
         {`Beta feature: Below is the shared word card`}
       </Typography>
-      <Stack minWidth={700}>
+      <Stack minWidth={700} style={{ border: `3px solid red` }}>
```
Comment (mlajkim) [inline src/pages/share/index.tsx:29]: Please remove these after test is done

**Review by mlajkim — COMMENTED** [review]
First review

File: src/components/molecule_word_card/index.shared.tsx (hunk 1 above)
Comment (akanex115) [reply src/components/molecule_word_card/index.shared.tsx]: OK. Handled in [2192054](https://github.com/ajktown/wordnote/pull/172/commits/21920546e52a48eee1106073782d08c2b037d830)

File: src/pages/share/index.tsx (hunk 2 above)
Comment (akanex115) [reply src/pages/share/index.tsx]: Sorry. Handled in [33e9745](https://github.com/ajktown/wordnote/pull/172/commits/33e9745905a09891f7c9a8995826f8d09e372030)

**Review by mlajkim — APPROVED** [review]
OK

**Pull request merged**